nixos/modules/security/pam.nix

```diff
@@ -253,6 +267,8 @@ let
               "auth sufficient ${pkgs.pam_u2f}/lib/security/pam_u2f.so"}
           ${optionalString cfg.usbAuth
               "auth sufficient ${pkgs.pam_usb}/lib/security/pam_usb.so"}
+          ${optionalString cfg.googleAuth
+              "auth required ${pkgs.googleAuthenticator}/lib/security/pam_google_authenticator.so"}
```

Comment:
AFAIK, this won't work due to `pam_deny.so` being `required` a little farther down the stack.
I've been putting off on making this PR myself until I got around to cleaning up our PAM module, as I don't think `requisite`, `sufficient`, or `optional` work here either.
(Specifically, we want this to work in the case someone has configured e.g 3FA for SSH - regular SSH authentication, then Google Authenticator, then something else).